example_config.py

```diff
@@ -0,0 +1,6 @@
+def main():
```

Comment:
Will merging of this clobber the work done in PR#6?

Reply:
What should I do to fix it?
Do I just put everything in ```py
 if __name__ == "__main__":
 ```

Reply:
No; I think this should be fine.